src/umpire/resource/FileMemoryResource.hpp

```diff
@@ -36,6 +36,7 @@ class FileMemoryResource :
 
   private:
     util::MemoryMap<std::pair <const std::string, std::size_t>> m_size_map;
+    int FILE_COUNTER = 0;
```

Comment:
Need to allow for potentially multiple object instantiations within the same process.
```suggestion
    static int FILE_COUNTER;
```

This will need to be initialized in the FileMemoryResource.cpp just inside of the `namespace` declarations as so:

`int FileMemoryResource::FILE_COUNTER{0};`